src/runtime-rs/config/configuration-dragonball.toml.in

```diff
@@ -91,7 +91,7 @@ default_bridges = @DEFBRIDGES@
 default_memory = @DEFMEMSZ@
 
 # Block storage driver to be used for the hypervisor in case the container
-# rootfs is backed by a block device. DB only supports virtio-blk.
+# rootfs is backed by a block device. dragonball only supports virtio-mmio.
 block_device_driver = "@DEFBLOCKSTORAGEDRIVER_DB@"
```

Comment:
Does dragonball support virtio-pci devices at all?

Reply:
Hi there, we have support for virtio-pci but it's not open-sourced yet.